### PR TITLE
Add PERP to optimism

### PR DESCRIPTION
**Pull request by 42bchen**
Description :

- Added PERP L1 and L2 token details to tokens list

### CODE REVIEW

**Comment by 42bchen** [comment]
@elenadimitrova 
hi, i got CI error when validate 0x08d32b0da63e2C3bcF8019c9c5d849d7a9d791e6 (DCN) symbol, do i need to do something in order to pass the CI check ?

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -891,7 +913,7 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
-	}
+    }
   ],
   "version": {
     "major": 3,
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:919]: Can you increment the patch number her as well.

File: optimism.tokenlist.json [hunk 2]
```diff
@@ -774,6 +774,28 @@
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
     },
+    {
+      "chainId": 1,
+      "address": "0xbC396689893D065F41bc2C6EcbeE5e0085233447",
+      "name": "Perpetual",
+      "symbol": "PERP",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/PERP.png",
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:783]: Add this image to the PR with roughly size 256x256 , you can use a smaller version but preferably not larger.

**Comment by elenadimitrova** [comment]
@42bchen thanks for reporting. I have fixed CI here https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/53 and left you a couple of small fixes here.

File: optimism.tokenlist.json (hunk 2 above)
Comment (42bchen) [reply optimism.tokenlist.json:783]: The image has added in previous PR (https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/51)

**Comment by 42bchen** [comment]
Done : 
- rebase to master
- increase the patch number from `1` to `2`

File: optimism.tokenlist.json (hunk 2 above)
Comment (elenadimitrova) [reply optimism.tokenlist.json:783]: Ah yes sorry.

**Review by elenadimitrova — APPROVED** [review]

**Pull request merged**